RUFAS/routines/field/manager/field_manager.py

```diff
@@ -87,9 +88,12 @@ def daily_update_routine(self, weather: Weather, time: Time) -> None:
                 "units": MeasurementUnits.HOURS,
             }
             self.om.add_variable("daylength", current_conditions.daylength, info_map)
-            field.manage_field(time, current_conditions=current_conditions)
+            crops = field.manage_field(time, current_conditions=current_conditions)
```

Comment:
```suggestion
            harvested_crops = field.manage_field(time, current_conditions=current_conditions)
```
I think this makes the whole harvesting process easier to track in the S&C module.

Reply:
agreed

Reply:
seconding this suggestion.

Reply:
In this particular instance, I changed `crops` to `newly_harvested_crops`, because they are the arg to the `extend` call on `harvested_crops`. But I have changed every other spot in the Field module to use `harvested_crops` as the name.